ci/ci/ci.py

```diff
@@ -203,15 +203,15 @@ def refresh_deploy_jobs(jobs):
         else:
             if job_ordering(job, job2) > 0:
                 log.info(
-                    f'cancelling {job2.id}, preferring {job.id}'
+                    f'deleting {job2.id}, preferring {job.id}'
                 )
-                try_to_cancel_job(job2)
+                try_to_delete_job(job2)
```

Comment:
I feel like cancelling running deploy jobs is not good.  Should we only do this if it is complete?  Maybe it doesn't really matter since this shouldn't happen.

Reply:
we get here if there are two deploy jobs running. It seems like we *definitely* shouldn't have two deploy jobs running.

Reply:
Yes, that's true.  Damned if you, etc.